checks/raw/security_policy.go

```diff
@@ -29,23 +30,38 @@ import (
 )
 
 type securityPolicyFilesWithURI struct {
-	uri   string
-	files []checker.File
+	info []checker.SecurityPolicyInformation
+	uri  string
+	file checker.File
```

Comment:
You may have to make fields fields public, so that we can access them from `pkg/` folder.

Reply:
if you are referring to `securityPolicyFilesWithURI`, that struct (internal to `checks/raw/security_policy.go`) were NOT public prior to my changes for this PR

```
type securityPolicyFilesWithURI struct {
        uri   string
        files []checker.File
}

if you want all those fields public from this point on, please let me know and I will make them public---I was only following the pattern I saw when I started this PR.
```

Reply:
closing this comment for now - we can revisit if necessary